.github/ISSUE_TEMPLATE/performance-question.md

```diff
@@ -0,0 +1,29 @@
+---
+name: "\U0001F947 Performance Question"
+about: Performance question about TiDB which is not caused by bug.
+
+---
+
+## Performance Question
+
+- What version of TiDB are you using?
+<!-- You can try `tidb-server -V` or run `select tidb_version();` on TiDB to get this information -->
+
+- What's the difference between the performance you expect and the performance you observe?
+
+- Are you comparing TiDB with some other databases? What's the difference between TiDB and the compared one?
+
+- If it's a specific SQL query, could you please
+    - let us know whether you analyzed the tables involved in the query or not. And how long it is after you ran the last `ANALYZE`.
```

Comment:
s/how long it is/how long does the sql use ?

Reply:
No. It's <code>how long it takes from the time of your last `ANALYZE`</code>.